packages/commonwealth/client/scripts/views/Sublayout.tsx

```diff
@@ -38,37 +38,34 @@ const Sublayout = ({
   }, [forceRerender]);
 
   useEffect(() => {
-    let timer;
+    setMenu({ name: 'default', isVisible: !isWindowSmallInclusive });
+  }, [isWindowSmallInclusive]);
```

Comment:
React Hook useEffect has a missing dependency: 'setMenu'. Either include it or remove the dependency array